src/context/product/ProductReducer.js

```diff
@@ -0,0 +1,26 @@
+export const reducerFn = (state, action) => {
+  switch (action.type) {
+    case "SORT":
+      return {
+        ...state,
+        sort: action.payload || ""
```

Comment:
Inside this payload the value which coming up what is that is it a id, category ,item  etc you can make payload as object and give whatever property you want .
ex : action.payload.item this makes more sense that inside payload item is coming up .

Reply:
it is a string "increasing" or "decreasing" and depending on that my sort works.